src/components/Office/PPM/PPMHeaderSummary/PPMHeaderSummary.stories.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Grid, GridContainer } from '@trussworks/react-uswds';
+
+import PPMHeaderSummary from './PPMHeaderSummary';
+
+export default {
+  title: 'Office Components / PPM / PPM Header Summary',
+  component: PPMHeaderSummary,
+  decorators: [
+    (Story) => (
+      <GridContainer>
+        <Grid row>
+          <Grid col desktop={{ col: 2, offset: 8 }}>
+            <Story />
+          </Grid>
+        </Grid>
+      </GridContainer>
+    ),
+  ],
+};
+
+const Template = (args) => <PPMHeaderSummary {...args} />;
+
+export const WithAdvance = Template.bind({});
+WithAdvance.args = {
+  ppmShipment: {
+    actualMoveDate: Date.now(),
```

Comment:
See the above comment about `PPMShipmentShape`. Hard-coding these dates should also prevent Happo diffs when they are rerun against this.

```suggestion
    actualMoveDate: '02-Dec-22',
```